enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -341,185 +335,303 @@ risks and disruption when rolling out changes to their environments.
+Establish a new namespaced Custom Resource Definition (CRD) called `ChangeManagementPolicy` which allows cluster lifecycle
+administrators to capture their requirements for when resource(s) associated with the policy can initiate material 
+changes on the cluster.
+
+Add a `changeManagement` stanza to several existing resources in the OpenShift ecosystem which can reference
+the new `ChangeManagementPolicy` resource to restrict when and how their associated controllers can initiate
+material changes to a cluster.
+- HCP's `HostedCluster`. Honored by HyperShift Operator and supported by underlying CAPI providers.
+- HCP's `NodePool`. Honored by HyperShift Operator and supported by underlying CAPI providers.
 - Standalone's `ClusterVersion`. Honored by Cluster Version Operator.
 - Standalone's `MachineConfigPool`. Honored by Machine Config Operator.
 
-The implementation of `changeManagement` will vary by profile
-and resource, however, they will share a core schema and provide a reasonably consistent user
-experience across profiles. 
+Changes that are not allowed to be initiated due to a change management policy will be 
+called "pending". Controllers responsible for initiating pending changes will await a permissive window 
+according to each resource's relevant `changeManagement` configuration.
 
-The schema will provide options for controlling exactly when changes to API resources on the 
-cluster can initiate material changes to the cluster. Changes that are not allowed to be
-initiated due to a change management control will be called "pending". Subsystems responsible
-for initiating pending changes will await a permitted window according to the change's
-relevant `changeManagement` configuration(s).
+In additional to "policies", different resource kinds may offer additional knobs in their `changeManagement`
+stanzas to provide cluster lifecycle administrators additional control over the exact nature of 
+the changes desired by the resource's associated controller. For example, in `MachineConfigPool`'s 
+`changeManagement` stanza, a cluster lifecycle administrator will be able to (optionally) specify the 
+exact rendered configuration the controller should be working towards (during the next permissive window) vs
+the traditional OpenShift model where the "latest" rendered configuration is always the destination. 
+
+### ChangeManagementPolicy Resource
 
-### Change Management Strategies
-Each resource supporting change management will add the `changeManagement` stanza and support a minimal set of change management strategies.
-Each strategy may require an additional configuration element within the stanza. For example:
 ```yaml
+kind: ChangeManagementPolicy
+metdata:
+  # ChangeManagementPolicies are namespaced resources. They will normally reside in the 
+  # namespace associated with the controller initiating material changes. 
+  # For example, in Standalone namespace/openshift-machine-config for the MCO and 

```

Comment:
nit: there's no `openshift-machine-config` namespace; all the MCO components are in `openshift-machine-config-operator`.  E.g. in [a 4.17.0-ec.1 CI run][1]:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-ovn-serial/1806673110347288576/artifacts/e2e-aws-ovn-serial/gather-extra/artifacts/namespaces.json | jq -r '.items[].metadata.name' | grep -1 machine-config
openshift-machine-api
openshift-machine-config-operator
openshift-marketplace
```

[1]: https://prow.ci.openshift.org/view/gs/test-platform-results/logs/periodic-ci-openshift-release-master-nightly-4.17-e2e-aws-ovn-serial/1806673110347288576

Reply:
Thanks @wking - update in next push.